### PR TITLE
Improve error reporting

### PR DESCRIPTION
**Pull request by stdavis**
## Description of Changes
This pull request improves error reporting on email and console reports. Specifically, it adds the machine name to global error emails and shows copy errors in ship emails and console reports.

I'm not sure if these need to be addressed in the slack reports.

image
image
image

### Test results and coverage

```
```

### Speed test results

```
```

### CODE REVIEW

**Comment by steveoh** [comment]
👍 This should be added to the slack report as well.

**Comment by stdavis** [comment]
image

File: src/forklift/slack.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def lift_report_to_blocks(report):
         if _safely_access(pallet, 'success'):
             success = ':heavy_check_mark:'
 
-        message.add(SectionBlock(f'{success} *{_safely_access(pallet, "name").split(":")[2]}*'))
+        message.add(SectionBlock(f'{success} *{_safely_access(pallet, "name").split(":")[1]}*'))
```
Comment (steveoh) [inline src/forklift/slack.py]: are you sure these 2s should be 1s?

Comment (stdavis) [reply src/forklift/slack.py]: ¯\_(ツ)_/¯ I wasn't sure why, but it was throwing an error on my machine. I was going to ask you about it.

Comment (steveoh) [reply src/forklift/slack.py]: pretty sure you'll have to revert this.

pallet names on a normal windows machine will be `c:\path\to\pallet:name`.

**Pull request merged**